src/main/java/net/joshka/junit/json/params/JsonConverter.java

```diff
@@ -1,18 +1,23 @@
 package net.joshka.junit.json.params;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.json.JsonObject;
 import org.junit.jupiter.api.extension.ParameterContext;
 import org.junit.jupiter.params.converter.ArgumentConversionException;
 import org.junit.jupiter.params.converter.ArgumentConverter;
 
-import jakarta.json.JsonObject;
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 public class JsonConverter implements ArgumentConverter {
+    private static final Logger logger = Logger.getLogger(JsonConverter.class.getSimpleName());
```

Comment:
What's the impact of adding logging to the tests? Does it get in the way of the output / cause any problems?

I suspect that it might be worth removing this to avoid problems with code that also logs. (But again, it's been a while since I looked at Java and can't recall where J.u.l fits in to things).